docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -187,21 +189,52 @@ REST API and connect to a local validator:
 
   $ sudo -u sawtooth sawtooth-rest-api -v
 
-Running a Transaction Processor
-===============================
+Starting and Configuring the Transaction Processors
+===================================================
+
+This section describes how to start the IntegerKey and Settings family
+transaction processors, confirm that the REST API is running, and tell
+the validator or validator network to accept transactions from these
+transaction families.
 
 Transaction processors can be started either before or after the validator is
 started.
 
-The IntegerKey transaction processor is provided as a simple example of a
-transaction family, which can also be used for testing purposes.
+Supported Transaction Families
+------------------------------
+
+Sawtooth supports multiple languages for transaction processor development and
+includes additional transaction processors written in several languages.
+Sawtooth includes following transaction processors:
+
+* settings-tp - A Settings family transaction processor written in Python
+
+* intkey-tp-go - An IntegerKey transaction processor written in Go
+
+* intkey-tp-java - An IntegerKey transaction processor written in Java
+
+* intkey-tp-javascript - An IntegerKey transaction processor written in
+  JavaScript (requires node.js)
+
+* intkey-tp-python - An IntegerKey transaction processor written in Python
+
+* poet-validator-registry-tp - A transaction family used by the PoET consensus
+  algorithm implementation to keep track of other validators
+
+* xo-tp-javascript - An XO transaction processor written in JavaScript
+  (requires node.js)
+
+* xo-tp-python - An XO transaction processor written in Python
 
 .. note::
 
   In a production environment, you should always run a transaction processor
-  that supports the config transaction family.
-  See `Configuring the List of Transaction Families`_ for more information.
+  that supports the Settings transaction family.
```

Comment:
and the Identity transaction family.

Reply:
Ditto (will fix this for all three tutorials).